backend/sites/src/app/dto/response/response.dto.ts

```diff
@@ -1,9 +1,12 @@
-import { Type } from '@nestjs/common';
 import { Field, Int, ObjectType } from '@nestjs/graphql';
```

Comment:
Also i just noticed that we have one more  dto  (baseHttpResponse) with the same fields, can we look at re-using it? or is there a reason for this new one ?